modules/http4s/build.sbt

```diff
@@ -4,7 +4,7 @@ name := "pureconfig-http4s"
 
 crossScalaVersions := Seq(scala212, scala213)
 
-libraryDependencies ++= Seq("org.http4s" %% "http4s-core" % "0.21.21")
+libraryDependencies ++= Seq("org.http4s" %% "http4s-core" % "1.0.0-M20")
```

Comment:
We don't usually publish modules for milestone versions of upstream dependencies. Can we delay this update for when a stable release of http4s in the 1.0 series comes out?

Reply:
Downgraded version. Please see my response in different conversation for what it means for compatibility between modules.